src/sentry/static/sentry/app/components/avatar/sentryAppAvatar.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import BaseAvatar from 'app/components/avatar/baseAvatar';
+
+class SentryAppAvatar extends React.Component {
+  static propTypes = {
+    sentryApp: PropTypes.object.isRequired,
```

Comment:
Can we add a `shape` into `app/sentryTypes` for this? It doesn't need to include everything since this is a WIP. It will help others understand what type of object to expect.